### DISCUSSION TITLE
Regarding Custom Resolution

### DISCUSSION THREAD

**Discussion by sualehumer** [community/community#36858] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not a developer or anything, but i am in love with your custom resolution tweak, i use it everyday cuz it saves me tons of battery power.. the only thing i'd wish for is maybe a one button widget i can put on my screen to switch between the highest and the lowest resolution with one tap!

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're trying to contact the developer of some software hosted on GitHub, but the community is mostly for discussion around Git and GitHub. You should look in the *repository for that software* for how to reach the developers. In many cases an issue or discussion in that repository will work. :slightly_smiling_face:

**Comment by sualehumer** [comment]
Ok, i'll try find a proper place to send my feedback to the developer.. apologies for the mistake

**Reply by airtower-luna** [reply]
No problem! The main point is that you're much more likely to get a helpful response in the right place. :wink:

**Comment by patronbenim** [comment]
Merhaba arkadaşlar ben geliştirici değilim kullandığım uygulama hakkında yorum yapmak isterim YouTube uygulaması kullanıyorum telefonum Android 12 güncellemesi sonrası uygulama geri tuşu hatası veriyor çıkamıyorum takılıp kalıyor
